compiler/ast/ast_parsed_types.nim

```diff
@@ -184,7 +184,7 @@ type
     pnkIteratorDef
     pnkMacroDef
     pnkTemplateDef
-    pnkTypeSection
+    pnkConstDef
```

Comment:
changes to this file don't seem necessary for this PR, I would split them out just to help shrink this one down.

Reply:
I need handle defs and sections separately, one def for one Suggest and one Section for multiple Suggests, I want make it easier in range detection.